src/lib/radon/ccdr/eth.ts

```diff
@@ -128,72 +144,84 @@ export const getCode = (address: EthAddress) => {
 export const getLogs = (filter: {
     fromBlock?: EthBlockHead,
     toBlock?: EthBlockHead,
-    address?: EthAddress | EthAddress[],
+    address?: EthAddress | EthAddress[] | Wildcard,
     topics?: Bytes32[],
-    blockHash?: Bytes32,
+    blockHash?: Bytes32 | Wildcard,
 }) => {
+    checkRpcWildcards(filter)
     if (filter?.blockHash && (filter?.fromBlock || filter?.toBlock)) {
-        throw new EvalError("CCDR: EthGetLogs: uncompliant use of 'blockHash'")
+        throw new TypeError("rpc.eth.getLogs: uncompliant use of 'blockHash'")
     }
     if (filter?.fromBlock) {
         if (!_isBlockHead(filter?.fromBlock)) {
-            throw new EvalError("CCDR: EthGetLogs: invalid 'fromBlock' value");
+            throw new TypeError("rpc.eth.getLogs: invalid 'fromBlock' value");
```

Comment:
Can you run the linter to delete all unnecessary semicolons?
```suggestion
            throw new TypeError("rpc.eth.getLogs: invalid 'fromBlock' value")
```